table.csv

```diff
@@ -429,3 +429,8 @@ holochain-key-v0,               holochain,      0x947124,       Holochain v0 pub
 holochain-key-v1,               holochain,      0x957124,       Holochain v1 public key + 8 R-S (63 x Base-32)
 holochain-sig-v0,               holochain,      0xa27124,       Holochain v0 signature  + 8 R-S (63 x Base-32)
 holochain-sig-v1,               holochain,      0xa37124,       Holochain v1 signature  + 8 R-S (63 x Base-32)
+fil-piece-unsealed,             serialization,  0xfi01,         Filecoin piece, raw data (CID = Piece Commitment)
+fil-sector-unsealed,            serialization,  0xfi02,         Filecoin sector, raw data (CID = Data commitment)
+fil-sector-sealed,              serialization,  0xfi03,         Filecoin sector, sealed and replicated (CID = Replication Commitment)
+fil-hash-unsealed,              multihash,      0xfi04,         Filecoin unsealed commitment hash (custom hashing alg)
+fil-hash-sealed,                multihash,      0xfi05,         Filecoin sealed commitment hash (custom hashing alg)
```

Comment:
What's the actual hashing algorithm? Or is this some kind of special merkle-tree based hash?

Reply:
according to @porcuquine, it is indeed custom. @porcuquine describes it as "SHA256 with a twist" -- different enough that he says it shouldn't just be SHA-256